src/street-utils.js

```diff
@@ -52,6 +52,21 @@ export function newScene(
   );
   geoLayer.setAttribute('data-no-transform', '');
 
+  // reset memory
+  const memoryEl = document.getElementById('memory');
+  if (memoryEl) {
+    memoryEl.removeAttribute('project-info');
+    memoryEl.setAttribute('project-info', '');
+  }
+
+  // reset AI Chat
+  if (
+    window.aiChatPanelRef &&
+    typeof window.aiChatPanelRef.resetConversation === 'function'
+  ) {
+    window.aiChatPanelRef.resetConversation();
+  }
+
```

Comment:
Wouldn't it better to use the newScene event to reset the conversion from the AIChatPanel component? Similar to what we do in src/editor/contexts/Geo.context.js and src/editor/index.js (search for newScene). Better to avoid any usage of global variables like this.

Reply:
Thanks good idea https://github.com/3DStreet/3dstreet/pull/1121/commits/ef7a077aca497a901b0ee0453aeaf3da7cbd332e